FND/Utilities/Analytics.rdf

```diff
@@ -738,6 +761,12 @@ For certain indices, one of the most common weighting factor is by market capita
 		<skos:definition>indicates the number of elements in a given population</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-utl-alx;hasReferenceDateTime">
+		<rdfs:label>has reference date and time</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+		<skos:definition>specifies a reference date and possibly a time for a given value</skos:definition>
```

Comment:
Circular definition. 
Also please avoid "specifies" in definitions. 
Need to clarify that's it's the time when the value was valid, as opposed to the time it was measured/sampled or published.
Finally I think this should be "the" not "a".

Reply:
Not sure what you are thinking here - the name of the class this restriction is on is NumericIndexValue, whose definition is "change in some aggregate relative to the value of the aggregate as of some reference date" - the property definition does not say anything about when it was measured or published.  We could delete this altogether and just use hasObservedDateTime if you prefer.

Reply:
we seem to be at cross purposes. My comment was on property hasReferenceDateAndTime and the fact that its definition "specifies a reference date...for a given value" was circular. The definition should be something like "the time when the value actually occurred"